io.github.electronstudio.WeylusCommunityEdition.json

```diff
@@ -0,0 +1,65 @@
+{
+  "id" : "io.github.electronstudio.WeylusCommunityEdition",
+  "runtime" : "org.freedesktop.Platform",
+  "runtime-version" : "23.08",
+  "sdk" : "org.freedesktop.Sdk",
+  "sdk-extensions" : [
+    "org.freedesktop.Sdk.Extension.rust-stable",
+    "org.freedesktop.Sdk.Extension.node18"
+  ],
+  "modules" : [ {
+    "name" : "WeylusCommunityEdition",
+    "buildsystem" : "simple",
+    "build-options": {
+      "append-path": "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm16/bin:/usr/lib/sdk/node18/bin",
+      "env": {
+        "RUSTFLAGS":"-A dead_code -A unused_imports -L /app/lib",
+        "CARGO_HOME": "/run/build/WeylusCommunityEdition/cargo",
+        "XDG_CACHE_HOME": "/run/build/WeylusCommunityEdition/flatpak-node/cache",
+        "npm_config_cache": "/run/build/WeylusCommunityEdition/flatpak-node/npm-cache",
+        "npm_config_nodedir": "/usr/lib/sdk/node18"
+      }
+    },
+    "build-commands" : [
+      "tar xvf x264-x264-4613ac3c15fd75cebc4b9f65b7fb95e70a3acce1.tar.bz2",
+      "cd x264-4613ac3c15fd75cebc4b9f65b7fb95e70a3acce1; ./configure --prefix=/app --exec-prefix=/app --disable-opencl --disable-cli --enable-static --enable-shared; make -j8; make install",
+      "tar xvf ffmpeg-6.1.1.tar.xz",
+      "cd ffmpeg-6.1.1 ; ./configure --prefix=/app --disable-debug --disable-programs --enable-gpl --enable-libx264  --disable-autodetect --enable-static --enable-shared; C_INCLUDE_PATH=/app/include LIBRARY_PATH=/app/lib make -j8; make install",
+      "sed -i 's/npm install/npm install --offline/g' build.rs",
+      "C_INCLUDE_PATH=/app/include LIBRARY_PATH=/app/lib cargo b --features ffmpeg-system --release",
+      "install -D target/release/weylus /app/bin/weylus",
+      "install -D weylus.desktop /app/share/applications/io.github.electronstudio.WeylusCommunityEdition.desktop",
+      "install -D io.github.electronstudio.WeylusCommunityEdition.metainfo.xml /app/share/metainfo/io.github.electronstudio.WeylusCommunityEdition.metainfo.xml",
+      "install -D io.github.electronstudio.WeylusCommunityEdition.svg /app/share/icons/hicolor/scalable/apps/io.github.electronstudio.WeylusCommunityEdition.svg"
```

Comment:
this looks like there should be a module a ffmpeg to build it normally. and one for x264.